### PR TITLE
PR review for alert component

### PR DESCRIPTION
**Pull request by manojkumar8400**
Create alert component.

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
looks good

File: documentation/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    style="width: 833px; height: 639px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (tanveertkd) [inline documentation/alert/alert.html:67]: you can move the styles to the common css file

**Review by tanveertkd — APPROVED** [review]
You might want to add some margin to the documentation body, The navigation bar is overlapping you document heading text. Otherwise this is good to go.

File: documentation/alert/alert.html (hunk 1 above)
Comment (manojkumar8400) [reply documentation/alert/alert.html:67]: Ok bro.

**Comment by manojkumar8400** [comment]
> looks good

Thank you.

**Comment by manojkumar8400** [comment]
> You might want to add some margin to the documentation body, The navigation bar is overlapping you document heading text. Otherwise this is good to go.

ok bro I will do that.

**Pull request merged**